### PR TITLE
feat: Implemented 'include-basis-data' flag

### PR DESCRIPTION
**Pull request by slb-halfspace**
# Description

This flag and SettlementReportArgs entry disables the basis-data related files from being packaged into the ZIP file generated by the SettlementReportJob. 

It does this by simply having both the Quarterly and Hourly time series tasks exit immidiately, meaning they never set their TaskValues which communicates to the Zip task which files to include. The zip task already handles the case of having zero files to zip, by throwing an exception. 

Until we implement the result-files into the job, I believe throwing an exception is the correct choice.

## Pull-request quality

- [ ] The title adheres to [this guide](https://github.com/Mech0z/GitHubGuidelines)
- [ ] Tests are written and executed locally
- [ ] Subsystem tests have been tested (by manually deploying to `dev_002`)

### CODE REVIEW

File: source/databricks/settlement_report/contracts/settlement-report-job-parameters-reference.txt [hunk 1]
```diff
@@ -16,4 +16,5 @@
 --energy-supplier-id=1234567890123
 --split-report-by-grid-area
 --prevent-large-text-files
+--skip-basis-data-csv-files
```
Comment (JesperGomes) [inline source/databricks/settlement_report/contracts/settlement-report-job-parameters-reference.txt:19]: I suggest we call it 'include-basis-data'. That is the name used in the .NET code. That also matches what the user does in the front-end.

Comment (slb-halfspace) [reply source/databricks/settlement_report/contracts/settlement-report-job-parameters-reference.txt]: Agreed, it has been changed to be called --include-basis-data and include_basis_data :)

File: source/databricks/settlement_report/settlement_report_job/domain/report_generator.py [hunk 2]
```diff
@@ -24,6 +24,8 @@ def execute_hourly_time_series(
     """
     Entry point for the logic of creating hourly time series.
     """
+    if not args.include_basis_data:
```
Comment (JesperGomes) [inline source/databricks/settlement_report/settlement_report_job/domain/report_generator.py:27]: maybe add a that asserts that dbutils.jobs.taskValues.get("hourly_time_series_files") and  dbutils.jobs.taskValues.get("quarterly_time_series_files")  is empty? 

See for instance test_execute_hourly_time_series__when_datahub_administrator_or_system_operator__returns_expected_number_of_files_and_content

This test also uses the taskvalues

Comment (slb-halfspace) [reply source/databricks/settlement_report/settlement_report_job/domain/report_generator.py:27]: Added a test to check the contents :)

**Review by JesperGomes — APPROVED** [review]

**Pull request merged**